go/oasis-node/cmd/debug/txsource/workload/runtime.go

```diff
@@ -38,7 +38,12 @@ const (
 	// Ratio of remove requests that should delete an existing key.
 	runtimeRemoveExistingRatio = 0.5
 
-	runtimeRequestTimeout = 120 * time.Second
+	// Since storage nodes are getting shutdown for longer periods of time and
+	// need to sync, had to increase the timeout to 3 minutes.
+	// XXX: can probably reduce back to 2 minutes once #3181 PR lands, which
```

Comment:
We will also need to configure runtime checkpointing in e2e/runtime/txsource so nodes can sync from checkpoints. cc @jberci